x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -262,6 +262,20 @@ describe('<IndexManagementHome />', () => {
       find('indexTableIndexNameLink').at(0).simulate('click');
     });
 
+    test('should be able to open a closed index', async () => {
+      const { find, actions } = testBed;
+      actions.clickManageContextMenuButton();
+
+      const contextMenu = find('indexContextMenu');
+      const openIndexButton = contextMenu
+        .find('button[data-test-subj="indexTableContextMenuButton"]')
+        .at(0);
+
+      await openIndexButton.simulate('click');
```

Comment:
I wonder if it's not this await that triggers the CI failure. Can you remove it and try?

Reply:
I removed that await locally and it fails locally still. My tests don't only fail in CI, they also fail locally. Are yours only failing in CI? @sebelga

Reply:
Sorry missed your comment in my emails. Apparently CI turned green so everything good now! 👍 😊

Reply:
Usually `await` before a `.simulate` can cause problems, they shouldn't be needed.